src/content/docs/configuration/sharing.mdx

```diff
@@ -93,4 +93,9 @@ pull_request_rules:
 
   - Values in the `default` key will be merged and remote default values will
     apply to local configuration.
+
+  - If there is a rule in any of the `pull_request_rules`, `queue_rules`, `priority_rules`, `partition_rules`
```

Comment:
```suggestion
  - If there is a rule in any of the `pull_request_rules`, `queue_rules`, `priority_rules`, `partition_rules`,
```

Reply:
Since there's a `or` after this word, i don't think a comma is here appropriate @DouglasBlackwood

Reply:
True in French, not in English 😄

Reply:
I was intrigued so i searched up a bit, and looks like both are valid syntax IIUC: https://www.grammarly.com/blog/comma-before-or/ @DouglasBlackwood

Reply:
Added the comma

Reply:
Interesting, thanks for the link.